deflect/SegmentParameters.h

```diff
@@ -69,6 +69,8 @@ struct SegmentParameters
 
     /** Is the image raw pixel data or compressed in jpeg format */
     bool compressed = true;
+
+    View view = View::mono; //!< Eye pass for the segment
```

Comment:
I had started like this but this breaks compatibility with older clients because the serialization code uses sizeof(SegmentParameters). I don't think a workaround is possible for all use cases without a proper protocol version exchange, which we don't have.